### DISCUSSION TITLE
Bug with overriding DisclosureContent styles

### DISCUSSION THREAD

**Discussion by ahcai** [twilio-labs/paste#1798] [Q&A] [answered]
Hi, I need to add some custom styles to the `DisclosureContent` component but because the custom props (style/className) are passed in before the hardcoded padding, it's not possible to override the styles: https://github.com/twilio-labs/paste/blob/main/packages/paste-core/components/disclosure/src/DisclosureContent.tsx#L31.

**Comment by ahcai** [comment]
Actually this seems to be the issue with a lot of the components: https://github.com/twilio-labs/paste/blob/main/packages/paste-core/components/table/src/Td.tsx#L8.

Also is there a reason `className` is one of the blocked props?

**Comment by kate-porter** [comment, marked as answer]
Hey @ahcai! Check out the [second question in our FAQ](https://paste.twilio.design/getting-started/faqs) on why we don't support classnames, and let us know if you have any additional questions on that. Happy to chat further.

Can you share more about what styles you are trying to override on Disclosure and why you want to override them? We do have a [Disclosure Primitive](https://paste.twilio.design/primitives/disclosure-primitive), which is a purely functional, unstyled Disclosure component that may suit your needs, but it would be helpful for us to understand your goals so we can help point you in the right direction.

**Reply by ahcai** [reply]
Hi @katieporter, that makes sense! I have a mock from design where the padding around the disclosure content is 0.

**Reply by kate-porter** [reply]
Ah, gotcha! I'd recommend checking in with your designer to see if that's necessary or if that might have been unintentional. If it's necessary for any reason, please do circle back with us and we can discuss further.
